app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy:
```

Comment:
You're comparing `masks_to_buy` to 0, but there's no need to do this. Python treats 0 as False, so you can simply write `if masks_to_buy:`.